magento/magento2
# Correct location of transparent.js

### Description (*)
The reference to this file was originally changed:
https://github.com/magento/magento2/commit/0ce439b1ecc2906dd840f37989c7c8ccf84dc6bf#diff-7fec090bf536a50e1bdfa20e039e27edR9
https://github.com/magento/magento2/commit/0ce439b1ecc2906dd840f37989c7c8ccf84dc6bf#diff-6de82c32cd46b3be45d4d65e403f4136R9

And the file was moved here:
https://github.com/magento/magento2/commit/0ce439b1ecc2906dd840f37989c7c8ccf84dc6bf#diff-b0845d35ccdd892fe5e1a93a9b714b66

The file move was not ported to 2.3 but the requirejs-config was, breaking
admin sales order create with PayflowPro and Authorize.net

### Fixed Issues (if relevant)
1. magento/magento2#19763

### Manual testing scenarios (*)
1. Configure credit card payments for Payflow Pro
2. Create order in admin panel
3. `adminhtml/Magento/backend/{{locale}}/Magento_Payment/js/transparent.js` should not return a 404 error.

### Contribution checklist (*)
